IQActionSheetPickerView/IQActionSheetPickerView.h

```diff
@@ -48,6 +48,11 @@ typedef NS_ENUM(NSUInteger, IQActionSheetPickerStyle) {
     IQActionSheetPickerStyleTimePicker,
 };
 
+/// Identifies an attributed string of the toolbar title for normal state.
+extern NSString * const kAttributesForNormalStateKey;
+/// Identifies an attributed string of the toolbar title for highlighted state.
+extern NSString * const kAttributesForHighlightedStateKey;
+

```

Comment:
Can you please do these renaming
kAttributesForNormalStateKey => kIQActionSheetAttributesForNormalStateKey
 kAttributesForHighlightedStateKey => kIQActionSheetAttributesForHighlightedStateKey

Because it's not a good idea to not prefix public extern variable in a library, it may conflict with other name inside project.